src/Nix/Effects.hs

```diff
@@ -24,6 +25,8 @@ class MonadFile m => MonadEffects m where
     getCurrentSystemOS :: m Text
     getCurrentSystemArch :: m Text
 
+    getPosixTime :: m (POSIXTime)
```

Comment:
We're not using this effect anymore right?

Reply:
@shlevy I'm currently using this effect in `Builtins.hs`. Should I not be using that there anymore?

Reply:
@shlevy Check out my latest changes when you get the chance. I will squash and rebase on your approval.